config/locales/user_mailer/en.yml

```diff
@@ -105,6 +105,16 @@ en:
       intro: Here are the details to verify your identity in person at a United States
         Post Office near you.
       subject: You’re ready to verify your identity with %{app_name} in person
+    in_person_verified:
+      greeting: Hello,
```

Comment:
The comp shows 'Hi <first name>,', but the user's first name isn't available to us when this email is rendered.